ORBIT/phases/design/scour_protection_design.py

```diff
@@ -105,33 +106,31 @@ def compute_scour_protection_tonnes_to_install(self):
 
         References
         ----------
-        .. [1] Det Norske Veritas AS. (2014, May). Design of Offshore Wind Turbine
-        Structures. Retrieved from
+        .. [1] Det Norske Veritas AS. (2014, May). Design of Offshore Wind
+        Turbine Structures. Retrieved from
         https://rules.dnvgl.com/docs/pdf/DNV/codes/docs/2014-05/Os-J101.pdf
-        """
+        """  # noqa: E501
```

Comment:
This is new, and it's because of a long line due to the math terms. This is another reason why I prefer 88 over 79 for line length because there were quite a few of these.